source/crud/tests/unified/countDocuments-comment.yml

```diff
@@ -0,0 +1,70 @@
+description: "count"
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false # Avoid setting fail points with multiple mongoses
+      uriOptions: { retryReads: false } # Avoid retrying fail points with closeConnection
```

Comment:
Are these two lines necessary? I don't see any fail points used in this file.

Reply:
No, they are not, thanks. Removed.